### PR TITLE
add default keymap for Prime_L V2

### PR DESCRIPTION
**Pull request by holtenc**

### CODE REVIEW

**Comment by holtenc** [comment]
Perhaps this layout for configurator should be separate from the original Prime_L because the bottom row key sizes have changed from V1 to V2

File: public/keymaps/primekb_prime_l_v2_default.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "keyboard": "primekb/prime_l_v2",
```
Comment (noroadsleft) [inline public/keymaps/primekb_prime_l_v2_default.json]:
```suggestion
  "keyboard": "primekb/prime_l/v2",
```

File: public/keymaps/primekb_prime_l_v2_default.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  "keymap": "default_8fe29f2",
```
Comment (noroadsleft) [inline public/keymaps/primekb_prime_l_v2_default.json]:
```suggestion
  "keymap": "default_42d6270",
```

File: public/keymaps/primekb_prime_l_v2_default.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  "commit": "8fe29f2784b1c14a75694a908b8a5116d9517b97",
```
Comment (noroadsleft) [inline public/keymaps/primekb_prime_l_v2_default.json]:
```suggestion
  "commit": "42d6270f28831e95d1cb9c14a7423d5b1d864d67",
```

**Comment by holtenc** [comment]
that should be done. at least in a round about way.

**Pull request merged**